src/iop/dual_demosaic.c

```diff
@@ -0,0 +1,255 @@
+/*
+    This file is part of darktable,
+    Copyright (C) 2010-2020 darktable developers.
```

Comment:
nitpicking: 2021

Reply:
For us over-60 years don't matter that much any more :-)